.github/workflows/release.yml

```diff
@@ -121,16 +121,9 @@ jobs:
         cross: [false]
         include:
           # Windows
-          - config: {os: windows-latest, target: x86_64-pc-windows-gnu}
-            toolchain: stable-x86_64-pc-windows-gnu
-            cross: false
           - config: {os: windows-latest, target: x86_64-pc-windows-msvc}
             toolchain: stable-x86_64-pc-windows-msvc
             cross: false
-          # Cross targets
-          - config: {os: ubuntu-latest,  target: x86_64-unknown-linux-musl}
```

Comment:
Why was this removed? I can't remember now

Reply:
It is not working well with the lto options for some of the dependencies.
So I decided to strip it for now, to at least have some working pipe until it is somehow solved.